indico/modules/events/abstracts/forms.py

```diff
@@ -390,30 +390,38 @@ class CreateEmailTemplateForm(EditEmailTemplateRuleForm):
                      "You can customize it later."))
 
 
-class AbstractForm(IndicoForm):
+class AbstractAttachmentForm(IndicoForm):
+    attachments = FileField(_('Attachments'), param_name='attachments', multiple_files=True)
+
+    def __init__(self, *args, **kwargs):
+        self.abstract = kwargs.pop('abstract', None)
+        self.event = kwargs.pop('event')
+        super(AbstractAttachmentForm, self).__init__(*args, **kwargs)
+
+
+class AbstractForm(AbstractAttachmentForm):
+    _order = ('title', 'description', 'submitted_contrib_type', 'person_links', 'attachments', 'submission_comments')
+
     title = StringField(_("Title"), [DataRequired()])
     description = IndicoMarkdownField(_('Content'), editor=True, mathjax=True)
     submitted_contrib_type = QuerySelectField(_("Type"), get_label='name', allow_blank=True,
                                               blank_text=_("No type selected"))
     person_links = AbstractPersonLinkListField(_("People"), [DataRequired()], default_author_type=AuthorType.primary)
     submission_comment = TextAreaField(_("Comments"))
-    attachments = FileField(_('Attachments'), param_name='attachments', multiple_files=True, lightweight=True)
 
     def __init__(self, *args, **kwargs):
-        self.event = kwargs.pop('event')
-        self.abstract = kwargs.pop('abstract', None)
+        super(AbstractForm, self).__init__(*args, **kwargs)
         description_settings = abstracts_settings.get(self.event, 'description_settings')
         description_validators = self._get_description_validators(description_settings)
         if description_validators:
             inject_validators(self, 'description', description_validators)
         if abstracts_settings.get(self.event, 'contrib_type_required'):
             inject_validators(self, 'submitted_contrib_type', [DataRequired()])
-        super(AbstractForm, self).__init__(*args, **kwargs)
```

Comment:
:warning:
I'm quite sure that this breaks things. `inject_validators` **must** be called before the super call to the wtforms bases